lib/rubocop/cop/rspec/rails/minitest_assertions.rb

```diff
@@ -14,6 +14,8 @@ module Rails
         #   refute_equal(a, b)
         #   assert_nil a
         #   refute_empty(b)
+        #   assert_true(a)
+        #   assert_false(a)
```

Comment:
I have already commented in detail, these are Test::Unit APIs, not Minitest APIs. It seems to contradict the description of "`Check if using Minitest matchers`".

Reply:
We don’t plan to introduce separate cops for teat::unit or rails-specific assertions. It handles all minitest-like assertions.

Reply:
I see, it would probably help to prevent confusion by mentioning such distinctions in the documentation.